pkgs/by-name/au/autobrr/package.nix

```diff
@@ -0,0 +1,96 @@
+{ lib, cacert, esbuild, jq, moreutils, buildGoModule, fetchFromGitHub, stdenvNoCC, nodePackages, typescript }:
```

Comment:
For such a long list of argument, it is customary to put one argument per line for easier diff reviewing.